.travis.yml

```diff
@@ -51,6 +51,24 @@ matrix:
         - docker run --rm --privileged multiarch/qemu-user-static:register --reset
         - docker run --rm -v `pwd`:`pwd` -w `pwd` "pqclean/debian-unstable-ppc" /bin/bash -c "uname -a &&
           make ${MAKETARGET}"
+    - name: "Run tests on qemu-arm32 (GCC)"
+      os: linux
+      services: docker
+      env:
+        - MAKETARGET="run-functest-all run-testvectors-all run-sanitizer-all run-symbol-namespace-all"
+      script:
+        - docker run --rm --privileged multiarch/qemu-user-static:register --reset
+        - docker run --rm -v `pwd`:`pwd` -w `pwd` "flowher/debian-buster-arm" /bin/bash -c "uname -a &&
```

Comment:
This should now be `pqclean/debian-buster-arm`

Reply:
yup, will change